pr_testing/test_multiple_prs.sh

```diff
@@ -1303,6 +1301,12 @@ if [ -z ${NO_POST} ] ; then
     if [ -d $LOCALRT/das_query ] ; then
       send_jenkins_artifacts $LOCALRT/das_query das_query/PR-${REPORT_H_CODE}/${BUILD_NUMBER}/PR || true
     fi
+    if [ -d $LOCALRT/profiling ]; then
+      send_jenkins_artifacts $LOCALRT/profiling/${CMSSW_IB}/${ARCHITECTURE}/${PROFILING_WORKFLOW}/ profiling/${CMSSW_IB}/${ARCHITECTURE}/${PROFILING_WORKFLOW}/
```

Comment:
@gartung , what will happen when more than one PR run profiling or pr tests run multiple times for the same PR?

Reply:
Updated.